.github/workflows/ci.yml

```diff
@@ -6,16 +6,20 @@ on:
       - main
   pull_request:
 
+env:
+  CARGO_TERM_COLOR: always
+  NIGHTLY_RUST_VERSION: nightly-2022-11-03
```

Comment:
I'm not sure of the reason why `nightly-2022-11-03` is used, but I kept this version anyway - if a bump is wanted I can make a separate PR, since that might break `clippy` and other tools.

Reply:
we inherited this rust version from the cairo repo and i haven't had time to update it. i think it is worth updating though